eventmesh-examples/src/main/java/org/apache/eventmesh/http/demo/sub/RemoteSubscribeInstance.java

```diff
@@ -66,7 +66,8 @@ private static void subscribeRemote() {
 
     private static void unsubscribeRemote() {
         final RequestParam subscribeParam = buildCommonRequestParam()
-            .addBody(SubscribeRequestBody.TOPIC, JsonUtils.toJSONString(Collections.singletonList(ExampleConstants.EVENTMESH_HTTP_ASYNC_TEST_TOPIC)))
+            .addBody(SubscribeRequestBody.TOPIC,
+                JsonUtils.toJSONString(Collections.singletonList(ExampleConstants.EVENTMESH_HTTP_ASYNC_TEST_TOPIC)))
             .addBody(SubscribeRequestBody.CONSUMERGROUP, ExampleConstants.DEFAULT_EVENTMESH_TEST_CONSUMER_GROUP)
```

Comment:
Is this line break added manually by you or is it an Soptless process that adds it? The length of this line didn't exceed the limit.

Reply:
It has been fixed in new commits.